### PR TITLE
User management box fix mulitple users

### PR DESCRIPTION
**Pull request by lemniscate613**
Different behavior on local and QC. Saving user with multiple roles was causing error in send_invite() helper function.

### CODE REVIEW

File: myjobs/models.py [hunk 1]
```diff
@@ -759,9 +759,14 @@ def send_invite(self, email, company, role_name=None, reason=""):
 
         if role_name:
             if settings.ROLES_ENABLED:
-                assigned_role = Role.objects.get(
-                    company=company, name=role_name)
-                user.roles.add(assigned_role)
+                # role_name could be an array like ["Admin", "PRM User"]
+                if isinstance(role_name, list):
+                    for role in role_name:
+                        user.roles.add(role)
```
Comment (tdphillips) [inline myjobs/models.py]: If role_name is actually a list of strings, you'll need to grab the roles from the database.

Comment (koonsies) [inline myjobs/models.py]: This can be done with user.roles.add(*roles)

Comment (lemniscate613) [inline myjobs/models.py]: Thank you @tdphillips.

**Comment by misterjonscott** [comment]
Provocative and entertaining.

**Pull request merged**